src/components/grid/grid.stories.mdx

```diff
@@ -408,72 +397,68 @@ This example is best viewed in the Canvas tab using full-screen mode with device
   </Story>
 </Preview>
 
-## GridContainer Props
+## Props
 
+### GridContainer 
```

Comment:
`GridContainer` styled-system `padding` props should be presented in a separate table like you've done for other components, see Accordion for an example.